prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -1138,11 +1139,15 @@ subroutine initProgramVariables(input, env)
     tSccCalc = input%ctrl%tScc
     tDFTBU = input%ctrl%tDFTBU
     tSpin = input%ctrl%tSpin
-    tREKS = .false.
-    if (tSpin) then
+    nSpin = 1
+    if (input%ctrl%reksIni%tREKS) then
+      ! REKS follows spin-restricted open-shell scheme so nSpin should be two in the main code, but
+      ! some variables such as qOutput should be treated in a restricted scheme. Here nSpin is set
+      ! to one and changes to two later in the initialization.
```

Comment:
I can not find the place, where `nSpin` changes to 2 in the initialization.

Reply:
@aradi `nSpin` changes to 2 in the `REKS_init` routine. You can find this at line 654 of reksvar.F90 file.